content/articles/nesting-navigators-and-routing-with-react-navigation-in-react-native/index.md

```diff
@@ -0,0 +1,266 @@
+Changing between screens is a requirement in nearly all mobile applications. React Native's `react-navigation` library is stunning and simple to use for this purpose. It's a well-known library for React Native application routing and navigation. 
+
+When you nest navigators, the screens of one navigator are rendered inside the screens of another. If there is a stack, switching to a different screen will cause a new screen to be displayed.
+
+Navigators are in charge of making the switch between different screens. There are numerous navigation types supported by React Navigation, including Stack, Drawer, and Tab Navigators. In addition to navigating between screens, it can be used to transfer information between them.
+
+Table of content:
+- [First Step- Application development and module installation](#first-step--application-development-and-module-installation)
+- [Second Step- Creating a HomeScreen and NewworkerScreen](#second-step--creating-a-homescreen-and-newworkerscreen)
+- [Third Step- Navigating React Pages using StackNavigator](#third-step--navigating-react-pages-using-stacknavigator)
+- [Fourth Step- Passing Data to Other Screens Using Context](#fourth-step--passing-data-to-other-screens-using-context)
+- [Fifth Step- Adding Buttons to HomeScreen and FriendsScreen](#fifth-step--adding-buttons-to-homescreen-and-friendsscreen)
+
+### Prerequisites
+You'll require the following supplies to follow along with this tutorial:
+
+1. An on-disk Node.js development environment. Make a Local Development Environment with Node.js and the [Installation Guide.](https://www.digitalocean.com/community/tutorial_series/how-to-install-node-js-and-create-a-local-development-environment)  
+
+2. An advantage for this project is familiarity with the iOS or Android simulators and setting up a new React Native environment in your development environment.
+
+There will be several steps that we will walk through to achieve our goal in this tutorial.
+
+### First Step- Application development and module installation
+
+1. Install expo-cli by running the following command in your terminal.
+
+```bash
+$ npm install -g expo-cli
+```
+
+2. The next step is to use the commands below to create a new project.
+
+```bash
+$ npx react-native init react-navigation-routing --version 0.63.2
+```

```

Comment:
Provide Docs for Setting up React Native Environment. Not sure why are you are installing Expo CLI and then creating a new project using the React Native CLI.